WordPress/gutenberg
# TextareaControl: Update styles

Part of #46741

## What?

Update styles on TextareaControl to match the current [specs](https://www.figma.com/design/804HN2REV2iap2ytjRQ055/Core-System-Library?node-id=2518-56299). Namely, increased padding and leading.

## Testing Instructions

See Storybook for TextareaControl. See usages in the editor.

Note for code review: The [first commit](https://github.com/WordPress/gutenberg/pull/64586/commits/d2f3c0749f23080b2eec23c6ba72287b51c87f34) is simply a move of the styles in `utils/input/` which were only used for this TextareaControl component. The [effective changeset](https://github.com/WordPress/gutenberg/pull/64586/files/d2f3c0749f23080b2eec23c6ba72287b51c87f34..27388897ac283cb2d9a51b391143e4c0d1ae469f) is much smaller.

## Screenshots or screencast 

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/6a0b97ee-f533-4725-bd9b-ecb4e7c0ca95" alt="TextareaControl, before" width="283">|<img src="https://github.com/user-attachments/assets/8b066ce9-e1ec-44f1-a85a-ef62b3cb01e7" alt="TextareaControl, after" width="283">|

How it looks in context with other controls:

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/c6d18ec2-11e0-41d8-9737-393223e133bb" alt="Image block inspector, before" width="274">|<img src="https://github.com/user-attachments/assets/30240d90-4c7d-49b4-aa84-e0d470efc44c" alt="Image block inspector, after" width="274">|

Or in this Additional CSS field:

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/3e890f9b-a223-41da-a721-f7cf3d171703" alt="Additional CSS input, before" width="274">|<img src="https://github.com/user-attachments/assets/bffe38f5-3669-4a29-88c1-284fdd3aeec2" alt="Additional CSS input, after" width="274">|
